eds/blocks/partners-navigation/features/profile/dropdown.js

```diff
@@ -40,6 +40,17 @@ const decorateProfileLink = (service, path = '') => {
   return `${serviceUrl}${path}`;
 };
 
+// PARTNERS_NAVIGATION START
```

Comment:
@slobodankajovicic we can't have this in the codebase. These URLs must be configured somewhere. how is it handled for DMe?